### ISSUE TITLE
GCP Connector service stopped working (network issue?)

### ISSUE THREAD

**Issue by jeffreyoelen** [google/cloud-print-connector#409]
Hi All, 

On our printserver (Windows Server) the GCP Connector is stopped working and wont start anymore, also not after installing version 1.15. During the install of version 1.15 were getting an arrow while it wants to start te service. We did not change anything on the machine so we're thinking that an network change stopped the service.

What could be network changes that cause stop the service from starting again? 

Error were getting during the intallation of version 1.15 is:
Service 'Cloud Print Connector' (Cloud Print Connector) failed to start. Verify that you have sufficient privileges to start system services. 

We did try to install version 1.15 from several Admin accounts with no success. Error keeps coming up during the installation when it wants to start the service. 

Here some screenshots:
https://drive.google.com/file/d/1hiFcqogJgHAVCQbfu5HZZhqxPKdzETIZ/view?usp=sharing
https://drive.google.com/file/d/12Bykg83xJugezS2I22vY9WbYVeltOKuw/view?usp=sharing

Hopefully you can help us out. 

Kind regards
Jeffrey

**Comment by jay0lee** [comment]
Can you try extracting the event viewer logs as described at:

https://github.com/google/cloud-print-connector/wiki/Windows-Event-Log-and-Troubleshooting

and attaching the generated connector.log file here? It should tell us why the service is failing to start.

**Comment by jeffreyoelen** [comment]
Hi Jay Lee,

Thanks for your quick reply.
I did ask the engineers to send me the log file, when I receive the file i
will attach it here for you.


Met vriendelijke groet / With kind regards,
[image: WEBR Logo]
------------------------------
*Jeffrey Oelen | Service Manager*
+31 6 2374 6067 |
------------------------------
 *G Suite* Certified Deployment Specialist
[image: Google Cloud Premier Partner]
Bent u klant van ons, meld u dan hier
<http://www.webr.nl/nieuwsbrief-inschrijven/> aan voor onze maandelijkse
nieuwsbrief!

2018-04-05 16:09 GMT+02:00 Jay Lee <notifications@github.com>:

**Comment by tentwelveeight** [comment]
We are having the same issue. See logs attached.

I've tried both 1.15 and 1.16. Another strange detail is that we have several virtualized print servers we would like to test the connector on. The first one I tried installed without issue, but the second throws the error. No difference between the two from what we can tell. Any advice is appreciated.

[GCPC_Event.txt](https://github.com/google/cloud-print-connector/files/1988887/GCPC_Event.txt)

**Comment by jeffreyoelen** [comment]
Hi @tentwelveeight do you still have this issue?

**Comment by tentwelveeight** [comment]
Yes. I just checked again this morning and the issue is still happening.

**Comment by jay0lee** [comment]
Log indicates XMPP connectivity issues. Check your firewall or switch to using (experimental) HTTPS-bases FCM support.
